config/routes.rb

```diff
@@ -126,6 +126,13 @@
 
   ################################################################################
   # UI
+
+  resources :webauthn_credentials, only: %i[index create destroy] do
+    collection do
+      get :create_options
```

Comment:
`get :create_options` route is not protected against CSRF. Consider if this should use `post` instead to gain CSRF protection.

Reply:
This action is not a state-changing action.

We renamed it, `create_options` was a bit confusing.